pkg/client/client.go

```diff
@@ -89,3 +89,14 @@ func NewClient(host string) (*Client, error) {
 		clientapi.DefaultSchemes, httpClient)
 	return &Client{*clientapi.New(clientTrans, strfmt.Default)}, nil
 }
+
+func improve(err error) error {
+	if err == nil {
+		return err
+	}
+	if strings.Contains(err.Error(), defaults.SockPath) {
+		return fmt.Errorf("Cannot connect to the Cilium Agent at unix://%s. Is the agent running?", defaults.SockPath)
```

Comment:
While I like the hint with regard to what could be wrong, we should continue to print the real error returned from the client as well. It could be permissions denied, no such file or directory, ...